source/plugins/wakatime/metadata.yml

```diff
@@ -53,3 +52,22 @@ inputs:
     type: number
     default: 5
     min: 0
+
+  # If you use a selfhosted wakatime instance
+  # that is publicly available set this to true
+  plugin_wakatime_wakapi:
+    description: Use WakaTime or self hosted Wakapi instance
+    type: boolean
+    default: no
```

Comment:
Can be deleted as we can merge into a single option with `plugin_wakatime_apiurl`